Doc/whatsnew/3.6.rst

```diff
@@ -238,8 +238,8 @@ and the ``__annotations__`` attribute.
       and Guido van Rossum. Implemented by Ivan Levkivskyi.
 
    Tools that use or will use the new syntax:
-   `mypy <http://github.com/python/mypy>`_,
-   `pytype <http://github.com/google/pytype>`_, PyCharm, etc.
+   `mypy <https://github.com/python/mypy>`_,
```

Comment:
The home is more http://www.mypy-lang.org/ no?

Reply:
Updated.